versioned/persist/serialize/src/main/proto/persist.proto

```diff
@@ -116,3 +116,8 @@ message RefPointer {
   Type type = 1;
   bytes hash = 2;
 }
+
+message RepoProps {
+  int32 repo_version = 1;
+  map<string, string> properties = 2;
```

Comment:
Does gRPC serialization guarantee a stable binary form of `map` entries?

Reply:
(See comment above.)